ph_logger.c

```diff
@@ -356,7 +356,6 @@ static int ph_logger_push_from_file(const char *filename, char *platform,
 				 * The reason being we can't differentiate between a slow
 				 * growing file and a file that doesn't grow at all.
 				 */
-				bytes_read = 0;
```

Comment:
unsure about this one too... It's inside a while and we check bytes_read in its condition

Reply:
Yes, but the next instruction is a `break` and `bytes_read` isn't check after the loop

Reply:
indeed